src/Cryptol/Symbolic/What4.hs

```diff
@@ -257,12 +260,22 @@ prepareQuery sym ProverCommand { .. } =
   simulate args =
     do let lPutStrLn = M.withLogger logPutStrLn
        when pcVerbose (lPutStrLn "Simulating...")
+
```

Comment:
This following sequence feels like a bit of a hack.  Binding the top-level decls of the module using `EWhere` does seem to work, but feels pretty dirty.  Any ideas how to handle this better, @brianhuffman, @yav?